README.md

```diff
@@ -139,6 +154,26 @@ __AWESOME__ apps counter: __111_ 🎉
 - [x] [GitHub](https://github.com/tachiyomiorg/tachiyomi)
 - [x] [Official page](https://tachiyomi.org/)
 
+### TachiyomiSY
```

Comment:
Please explain me the difference between `Tachiyomi️` and `TachiyomiSY`.

Reply:
> Please explain me the difference between `Tachiyomi️` and `TachiyomiSY`.

`More features`
[Read More Here] (https://github.com/jobobby04/TachiyomiSY)

Reply:
Holy smokes, how many forks are there? 😆

Reply:
Quite a bit, you can find most of them on the Tachiyomi website